scanner/src/routes/scan/+page.svelte

```diff
@@ -61,27 +63,39 @@
   onMount(() => {
     startScan()
   })
+
+  setTimeout(() => {
+    goto(`/loading`, {
+      replaceState: true,
+      state: {
+        dataspace_domain: "sandbox.ioxio-dataspace.com",
+        product_path: "DPP/Energy/Battery/ProductDataSheet_v0.1",
+        source: "dpp_demo",
+        product: "MPP48V",
+        id: "test",
+      },
+    })
+  }, 2000)
```

Comment:
Why do you think we want to move to this random loading page after 2 seconds on this page?

You should SCAN THE CODE, or use the `Simulate` button to go forward

Reply:
```suggestion
```